lib/game/player.ex

```diff
@@ -14,4 +14,9 @@ defmodule Game.Player do
   def create_player(player_type, symbol) do
     %Player{type: player_type, symbol: symbol}
   end
+
+  @spec get_move(Player.t(), map(), any) :: any
```

Comment:
Non-blocking comment – it would be nice if there were a `board` type rather than a `map` to improve typespec readability.